### ISSUE TITLE
Release v0.5.5

### ISSUE THREAD

**Issue by simon-mo** [vllm-project/vllm#7481]
We will make a release later this week or early next week (Aug 16-Aug19) to address Gemma logits soft-caps bug, openai server metrics bug, and include more performance enhancements. 

Please add blockers if needed.

**Comment by robertgshaw2-neuralmagic** [comment]
- [ ] https://github.com/vllm-project/vllm/pull/7279
- [ ] https://github.com/vllm-project/vllm/pull/7394

**Comment by njhill** [comment]
We should include https://github.com/vllm-project/vllm/pull/7424

**Comment by simon-mo** [comment]
- [ ] #7487

**Comment by robertgshaw2-neuralmagic** [comment]
- [ ] https://github.com/vllm-project/vllm/pull/7443

**Comment by simon-mo** [comment]
- [ ] #7415

**Comment by mgoin** [comment]
- [] https://github.com/vllm-project/vllm/pull/7534

**Comment by WoosukKwon** [comment]
- [ ] #7638 
- [ ] #7642

**Comment by youkaichao** [comment]
- [ ] https://github.com/vllm-project/vllm/pull/7659

**Comment by njhill** [comment]
- [ ] https://github.com/vllm-project/vllm/pull/7698

**Comment by robertgshaw2-neuralmagic** [comment]
Not required but nice + easy:
- [ ] https://github.com/vllm-project/vllm/pull/7716

**Comment by njhill** [comment]
A fix for https://github.com/vllm-project/vllm/issues/7632 (DoS issue)

**Comment by sindhuvahinis** [comment]
- [ ] A fix for phi-v3.5 model https://github.com/vllm-project/vllm/pull/7710

**Comment by Jimmy-Newtron** [comment]
> ... a release later this week or early next week (Aug 16-Aug19) ...

When do you plan the release now?

**Comment by simon-mo** [comment]
Now
